homework-submission/myapp/src/Components/Data.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Todos from './Todos.json';
+
+class Data extends React.Component {
+
+    state = {
+        Todos
+    }
+
+    isTaskDone = (todo) => {
+        todo.done = !todo.done;
```

Comment:
don't mutate the state directly IMPORTANT, you are mutating the state then using setState !!! it doen't make sense
make a deep copy and mutate then use set state

Reply:
Ohh!! my mistake. I have sent you a message on slack about this part... Thank you for the constructive feedback.